src/pages/DecipherPage/Redirection.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react'
```

Comment:
Remove importation of `useEffect` as it is not used to remove warning.

Reply:
Thanks